src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/ReactorAccessPort.java

```diff
@@ -26,64 +26,69 @@
 import me.mrCookieSlime.Slimefun.api.item_transport.ItemTransportFlow;
 
 public class ReactorAccessPort extends SlimefunItem {
-	
+
 	private static final int[] border = {0, 1, 2, 3, 4, 5, 6, 7, 8, 12, 13, 14, 21, 23};
 	private static final int[] border_1 = {9, 10, 11, 18, 20, 27, 29, 36, 38, 45, 46, 47};
 	private static final int[] border_2 = {15, 16, 17, 24, 26, 33, 35, 42, 44, 51, 52, 53};
 	private static final int[] border_3 = {30, 31, 32, 39, 41, 48, 49, 50};
 
 	public ReactorAccessPort(Category category, ItemStack item, String name, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, name, recipeType, recipe);
-		
+
 		new BlockMenuPreset(name, getInventoryTitle()) {
-			
+
 			@Override
 			public void init() {
 				constructMenu(this);
 			}
 
 			@Override
 			public void newInstance(BlockMenu menu, Block b) {
+				
+					if (BlockStorage.getLocationInfo(b.getLocation(), "reactor-port") == null){
+						BlockStorage.addBlockInfo(b, "reactor-port", "true");

```

Comment:
Why exactly is this tag addded to the block btw?

Reply:
Not exactly sure why I did that, must have tried as the port wouldn't open previously and forgotten to remove it.   Updated now should be good to go